pkg/client/results/processing.go

```diff
@@ -57,6 +57,18 @@ const (
 	// PostProcessedResultsFile is the name of the file we create when doing
 	// postprocessing on the plugin results.
 	PostProcessedResultsFile = "sonobuoy_results.yaml"
+
+	// metadataFileKey is the key used in an Item's metadata field when the Item is
+	// representing the a file summary (and its leaf nodes are individual tests or suites).
+	metadataFileKey = "file"
+
+	// metadataTypeKey is the key used in an Item's metadata field when describing what type
+	// of entry in the tree it is. Currently we just tag summaries, files, and nodes.
+	metadataTypeKey = "type"
```

Comment:
Is `type` the best word here? Open to other ideas, happy with type if you are though.

Reply:
The only thing I can think of is “source”, to indicate where a result came from, but I’m not particularly keen on that. I’m happy to stick with type.